alignak/daemon.py

```diff
@@ -146,12 +147,11 @@ def get_all_groups():
 
 
 class InvalidPidFile(Exception):
-    """Exception raise when a pid file is invalid"""
+    """Exception raised when a pid file is invalid"""
     pass
 
 
-DEFAULT_WORK_DIR = '/var/run/alignak/'
-DEFAULT_LIB_DIR = '/var/lib/alignak/'
+DEFAULT_WORK_DIR = './'
```

Comment:
This is not even useful because it is only used to initialize a property ... and when the property is used we os.chdir the current directory